stream_alert/rule_processor/threat_intel.py

```diff
@@ -401,3 +407,11 @@ def _deserialize(dynamodb_data):
     def normalized_type_mapping(cls):
         """Get normalized CEF types mapping to original keys from the records."""
         return cls.__normalized_types
+
+    @staticmethod
+    def is_public_ip(ip_address):
+        try:
+            ip_addr = IPAddress(str(ip_address))
+            return ip_addr.is_unicast() and not ip_addr.is_private()
```

Comment:
add a code comment about why you're checking if its unicast

Reply:
ex: do we not care about multicast?

Reply:
Good question. I just verified that. A multicast ip is a private ip. 
```
from netaddr import IPAddress

ip = IPAddress('239.192.0.1')
ip.is_multicast()
True

ip.is_unicast()
False

ip.is_private()
True
```